src/react/src/components/Contribute/ProductionLocationDialogCloseButton.jsx

```diff
@@ -8,22 +8,22 @@ import { makeProductionLocationCloseButtonStyles } from '../../util/styles';
 const ProductionLocationDialogCloseButton = ({
     isMobile,
     classes,
-    handleShow,
+    handleGoToMainPage,
 }) => (
     <>
         {isMobile ? (
             <IconButton
                 aria-label="Close"
                 className="mobile-dialog-close-button"
-                onClick={() => handleShow(false)}
+                onClick={() => handleGoToMainPage()}
             >
                 <CloseIcon />
             </IconButton>
         ) : (
             <IconButton
                 aria-label="Close"
                 className={classes.desktopCloseButton}
-                onClick={() => handleShow(false)}
+                onClick={() => handleGoToMainPage()}
```

Comment:
Since the handleGoToMainPage doesn't receive any parameters I recommend simplifying the construction:
```
onClick={handleGoToMainPage}

Reply:
Done.